src/clients/BundleDataClient/BundleDataClient.ts

```diff
@@ -1088,7 +1088,7 @@ export class BundleDataClient {
               deposit &&
               deposit.originChainId === originChainId &&
               deposit.destinationChainId === destinationChainId &&
-              deposit.blockNumber <= originBlockRange[0] &&
+              deposit.blockNumber >= originBlockRange[0] &&
```

Comment:
Is there a brief tldr on why this is needed btw?

Reply:
Oh sorry. I should move this commit to parent branch